src/patternfly/components/Alert/alert.scss

```diff
@@ -133,15 +148,16 @@
 
     // Set border left color in case it is also an inline alert
     --pf-c-alert--m-inline--before--BackgroundColor: var(--pf-c-alert--m-inline--m-info--before--BackgroundColor);
+
+    &.pf-m-inline {
+      --pf-c-alert__icon--Color: var(--pf-c-alert--m-inline--m-info__icon--Color);
+    }
   }
 
   &.pf-m-inline {
     --pf-c-alert--BoxShadow: var(--pf-c-alert--m-inline--BoxShadow);
     --pf-c-alert__icon--BackgroundColor: var(--pf-c-alert--BackgroundColor);
```

Comment:
Looks like we have a var for this value that is currently unused `--pf-c-alert--m-inline__icon--BackgroundColor`, and I think it should just be `transparent` instead of `var(--pf-c-alert--BackgroundColor)`.